modules/client/src/StateGenerator.ts

```diff
@@ -74,6 +75,18 @@ export function calculateExchange(args: ExchangeArgsBN) {
   }
 }
 
+export function depositIfNegative(r: any, src: string, dst: string) {
```

Comment:
Can we change `any` to something like `{ [key: string]: string }` or something similar?